examples/trials/NAS/macro_cifar10.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
```

Comment:
I think this PR's contribution comes from the general NAS interface.  

But for the RLtuner, you could see there are many hyper-parameters in RLTuner related to trials. I think users hard to set this parameter in tuners and also hard to achieve good performance as a general tuner. Other you could show us the experiment and experience to convince us.

 So I think we could move to other third-party repo as ENAS-nni, especially the example code.

Reply:
Sure, this is not a general tuner. It is just a specific one for the NAS trial code (I think that it is just like the ga_customer_tuner so I put this tuner here (in the `tuner` folder?)).

Reply:
@xuehui1991 I don't think the repo's name should include ENAS. The key reason is that the tuner could be used for NAS. Including ENAS would be a little misleading.